app/assets/stylesheets/wiki.css

```diff
@@ -68,9 +68,4 @@ a.navbar-offset {
     top: -100px;
     visibility: hidden;
 }
-@media (max-width: 768px) {
```

Comment:
Hi, just noting that when i look at these lines in blame view:

https://github.com/publiclab/plots2/blame/master/app/assets/stylesheets/wiki.css#L71

They're linked to this fix: 

https://github.com/publiclab/plots2/pull/3651

Perhaps rather than deleting it, we need to make the solution more specific to the original problem, with a narrower CSS selector. Then we don't have to undo the original fix?

Thank you!!!

Reply:
@jywarren after deleting this can we provide inline css to particular `Add` button ?